lib/data/models/sector.dart

```diff
@@ -1,30 +1,49 @@
 import 'package:flutter/foundation.dart';
 import 'package:kisgeri24/data/models/entity.dart';
 import 'package:kisgeri24/data/models/route.dart';
+import 'package:kisgeri24/data/models/wall.dart';
 
 class Sector extends Entity {
   String name;
-  List<Route> routes;
+  List<Wall>? walls;
+  List<Route>? routes;
 
-  Sector({
-    required this.name,
-    required this.routes,
-  });
+  Sector(this.name, this.walls, this.routes);
 
   static Sector fromSnapshot(String name, value) {
     Map placeMap = value as Map<dynamic, dynamic>;
     String placeName = name;
+    List<Wall> wallList = [];
     List<Route> routeList = [];
 
-    placeMap.forEach((key, value) {
-      final Route route = Route.fromSnapshot(value);
-      routeList.add(route);
-    });
-
-    Sector place = Sector(name: placeName, routes: routeList);
+    if (_sectorHasSubWall(value)) {
+      placeMap.forEach((key, value) {
+        final Wall route = Wall.fromSnapshot(key, value);
+        wallList.add(route);
```

Comment:
If the sector has a subwall, shouldn't this var be "wall" instead of "route"?

Reply:
that's right, gonna correct it, thanks! 🫸🏻🫷🏻

Reply:
@terjekig updated

Reply:
Perfect, thanks!